### PR TITLE
Adapt es6 modularization changes

### PR DESCRIPTION
**Pull request by normunds-ciq**

### CODE REVIEW

File: src/app/services/chart.service.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { Injectable, Optional } from '@angular/core';
 import { BehaviorSubject, ReplaySubject } from 'rxjs';
 import { map, filter, tap } from 'rxjs/operators';
 
-import { CIQ } from 'chartiq/js/chartiq';
-import { quoteFeedSimulator } from 'chartiq/examples/feeds/quoteFeedSimulator';
+import { CIQ } from 'chartiq/js/standard.js';
+import 'chartiq/js/advanced.js';
```
Comment (rbruckheimer) [inline src/app/services/chart.service.ts:6]: importing advanced is sufficient.

File: src/app/plugins/tfc.service.ts [hunk 2]
```diff
@@ -13,17 +14,15 @@ export class TfcService extends ITfc {
...
 			// @ts-ignore /* prevent compile failure when tfc plugin is not available as part of library */
 			import('chartiq/plugins/tfc/tfc-demo'),
-			import('chartiq/js/components'),
-		]).then(
-			([
-				{
-					CIQ: { TFC, Account },
-				},
-			]) => new TFC({
+			import('chartiq/js/webcomponents/sidepanel.js'),
```
Comment (rbruckheimer) [inline src/app/plugins/tfc.service.ts:17]: Importing individual file.  Can you guarantee its availability?

Comment (normunds-ciq) [reply src/app/plugins/tfc.service.ts]: Import replaced with know available file, comment provided for alternative if available

File: src/app/services/chart.service.ts (hunk 1 above)
Comment (normunds-ciq) [reply src/app/services/chart.service.ts]: Updated

File: src/app/plugins/tfc.service.ts (hunk 2 above)
Comment (rbruckheimer) [reply src/app/plugins/tfc.service.ts]: I cannot find the comment you provided.

**Comment by rbruckheimer** [comment]
Can you resolve conflict and test failure?

File: README.md [hunk 3]
```diff
@@ -1,53 +1,116 @@
...
+To enable the plug-in, uncomment the following lines of code:
+- *src/app.module.ts*
+  - // import { TfcService } from './plugins/tfc.service';
+  - // { provide: ITfc, useClass: TfcService },
+- *src/styles/scss*
+  - // @import '~chartiq/plugins/tfc/tfc.css';
+
+**Note:** Each line of code follows the comment: `// to enable tfc plugin uncomment next line`
+
+Open the Trade from the Chart panel by selecting the TFC control in the upper right corner of the chart.
```
Comment (samuel-ciq) [inline README.md:107]: This information appears to be incorrect:

1) There is no src/app.module.ts file. It is in src/app.
2) TFC is turned on by default in that file.

File: README.md [hunk 4]
```diff
@@ -1,53 +1,116 @@
...
+Any add-on can be included in your project by doing the following:
+1. Import the ChartIQ add-on source files (*chartiq/js/addOns*)
+2. Create and initialize an instance of the add-on
 
-## Enabling IE11
+For example, the [FullScreen](https://documentation.chartiq.com/CIQ.FullScreen.html) add-on creates a toggle control that enables the chart to occupy the entire display area of a device.
 
-- Run `npm install core-js` .
+To include the [FullScreen](https://documentation.chartiq.com/CIQ.FullScreen.html) add-on:
+1. Add the following static import to the top of *chart.service.ts*:
```
Comment (samuel-ciq) [inline README.md:68]: Can we add the path to this file?

File: src/app/app.module.ts [hunk 5]
```diff
@@ -25,7 +25,7 @@ import { ConfigService, loadConfig } from './services';
 
 import { ITfc } from './plugins/tfc.interface';
 // to enable tfc plugin uncomment next line
-// import { TfcService } from './plugins/tfc.service';
+import { TfcService } from './plugins/tfc.service';
```
Comment (samuel-ciq) [inline src/app/app.module.ts:28]: comment does not correspond to the uncommented line below

File: src/app/app.module.ts [hunk 6]
```diff
@@ -51,7 +51,7 @@ import { ITfc } from './plugins/tfc.interface';
 			multi: true,
 		},
 		// to enable tfc plugin uncomment next line
-		// { provide: ITfc, useClass: TfcService },
+		{ provide: ITfc, useClass: TfcService },
```
Comment (samuel-ciq) [inline src/app/app.module.ts:54]: same here re comment not matching next line

File: README.md (hunk 3 above)
Comment (normunds-ciq) [reply README.md]: Updated

File: README.md (hunk 4 above)
Comment (normunds-ciq) [reply README.md]: Updated

File: src/app/app.module.ts (hunk 5 above)
Comment (normunds-ciq) [reply src/app/app.module.ts]: Updated

File: src/app/app.module.ts (hunk 6 above)
Comment (normunds-ciq) [reply src/app/app.module.ts]: Updated

**Pull request merged**